src/clustering/extreme_vals.jl

```diff
@@ -168,7 +194,16 @@ function extreme_val_output(data::ClustData,
   else
     @error("rep_mod_method - "*rep_mod_method*" - does not exist")
   end
-  extr_vals = ClustData(data.region,K_ed,data.T,data_ed,weights_ed;mean=data.mean,sdv=data.sdv)
+  deltas_ed=data.deltas[:,unique_extr_val_idcs]
+  #QUESTION What with k_ids?
+  # if original time series period isn't represented by any extreme period it has value 0
+  k_ids_ed=zeros(Int64,size(data.k_ids))
+  index_k_ids_data=findall(data.k_ids.!=0)
+  k_ids_ed_data=k_ids_ed[index_k_ids_data]
+  # each original time series period which is represented recieves the number of it's extreme period in this extreme value output
+  k_ids_ed_data[unique_extr_val_idcs]=collect(1:K_ed)
+  k_ids_ed[index_k_ids_data]=k_ids_ed_data
```

Comment:
k_ids are included into the ClustData and need to be adjusted (to 0) if Extreme is selected.